### PR TITLE
Add readme, linterns and webpage files

### PR DESCRIPTION
**Pull request by sergiogval**
This is the first pull request to merge the first step for my portfolio project. It includes linterns settings, a readme file and the proper webpage files.

### CODE REVIEW

**Review by dasileker — COMMENTED** [review]
### **Hi @sergiogval** 


### **Project Invalid ❌❌**
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that you have an issue in your linters so you [go to this link](https://github.com/meronokbay/microverse-linters-setup) and redo the installation of the linter


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,65 +1,61 @@
...
 
-# Project Name
+# My Portfolio
 
-> One paragraph statement about the project.
+> First step to develop my own portfolio website .
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
```
Comment (DanteAlonsoHT) [inline README.md:10]: I know you did a good effort doing this description for your project, but just try to make a more descriptive README file, if you want to have a reference, three lines or more would be nice, just try to think about what you did, what kind of technologies or techniques you used, and so on. Keep in mind this for this and your next projects 😉 

- [ ] Improve description for your project.

File: README.md [hunk 2]
```diff
@@ -1,65 +1,61 @@
...
 
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### Authors
 
-👤 **Author2**
+👤 **Author**
```
Comment (DanteAlonsoHT) [inline README.md:48]: - [ ]  [OPTIONAL] Intead `👤 **Author**` you can use `👤 **Your_Name**` to make a more professional README file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <p class="text3">I can help you build a product, <br> feature or website. Look through <br> some of my work and experience! <br>
+      If you like what you see and have <br> a project you need coded, <br> don't hestiate to contact me.</p>
+  </section>
+  <div class="container">
+    <img src="images/tweet.png" alt="tweet" class="icon"/>
+    <img src="images/linkedin.png" alt="linkedin" class="icon"/>
+    <img src="images/github.png" alt="github" class="icon"/>
+    <img src="images/medium.png" alt="medium" class="icon"/>
+    <img src="images/angelist.png" alt="angelist" class="icon"/>
+  </div>
```
Comment (DanteAlonsoHT) [inline index.html:36]: You should add an `a` tag for each one social media icon you have, you can add the like following the next code:
``` 
<a href="#">
  <img src="" alt="">
</a>
```

- [ ] Add `a` tag for all your social media icons.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫♻️ </h1>

Hi @sergiogval, 

You did an amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
<hr>

I know you have the capabilities enough to finish this project! 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sergiogval) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (sergiogval) [reply README.md]: done

File: index.html (hunk 3 above)
Comment (sergiogval) [reply index.html]: done

**Review by mke2111 — APPROVED** [review]
### Hi @sergiogval, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**